src/Mod/Fem/femmesh/meshtools.py

```diff
@@ -2508,4 +2508,21 @@ def compact_mesh(
     # may be return another value if the mesh was compacted, just check last map entries
     return (new_mesh, node_map, elem_map)
 
+# ************************************************************************************************
+def beam_reduced_integration(
```

Comment:
Could you please reformat to use four-space indentation? Thanks!

Reply:
> Could you please reformat to use four-space indentation? Thanks!

Sure, done.